### DISCUSSION TITLE
Hide this Message on Espanso start.

### DISCUSSION THREAD

**Discussion by Dtrieb** [espanso/espanso#1534] [Feature requests and Ideas]
I would like it, if I only hide this message on the Espanso start.

![Espanso](https://user-images.githubusercontent.com/3650284/220709643-6894187c-94cc-414e-9579-56429e283a7c.png)

**Comment by smeech** [comment]
Sorry about the late reply - I am trying to mop up and sort the old Issues and Discussions.

You've probably worked it out by now, but you can set `show_notifications: false` in your `$CONFIG/config/default.yml` file, as documented [here](https://espanso.org/docs/configuration/options/#hiding-the-notifications).

Is that what you mean?
